cluster/config-defaults.yaml

```diff
@@ -28,6 +28,9 @@ cluster_autoscaler_max_graceful_termination_sec: "1209600" # 2 weeks
 cluster_autoscaler_max_usnchedulable_pods_considered: "1000"
 
 # karpenter settings
+# DO NOT SET TO FALSE IF THE CLUSTER HAS KARPENTER POOLS OR NODES. REFER TO TEAPOT DOCS FOR HOW TO ROLLBACK KARPENTER
```

Comment:
Do we already have the docs created? I know that the scripting is in progress, but was wondering if there's any doc PR. We should link that here.

Reply:
yea, here
https://github.bus.zalan.do/teapot/docs/pull/438

will add the link to the docs page

Reply:
Let's add the doc link here and then we can merge :)